arch/arm64/core/Kconfig

```
@@ -193,6 +193,15 @@ config EXCEPTION_DEBUG
 config XIP
 	select AARCH64_IMAGE_HEADER
 
+config SET_VMPIDR_EL2
+	bool "Set VMPIDR_EL2 at EL2 stage"
+	default n
+	help
+	  The purpose of VMPIDR_EL2 is that holds the value of the
+	  Virtualization Multiprocessor ID and this is the value returned
+	  by EL1 reads of MPIDR_EL1.
+	  Normally VMPIDR_EL2 is configurated at EL2 stage.
```

Didn't want to block merging for this, but since there are already some other change requests (thanks @stephanosio  for catching those), this should probably be updated for clarity:

```suggestion
	  VMPIDR_EL2 holds the value of the Virtualization Multiprocessor
	  ID. This is the value returned by EL1 reads of MPIDR_EL1.
	  Normally VMPIDR_EL2 is configured at the EL2 stage.
```

I'm not sure I follow the last line, though, since `SET_VMPIDR_EL2` defaults to `n`,
implying that this isn't 'normally' being set at the EL2 stage. (My experience is more
with Cortex-M, though, so I'm not as familiar with the registers here as I perhaps
should be to comment. The naming and comments aren't helping me, though.)

---

@carlocaione Can you perhaps clarify or propose a clearer explanation of what this config setting is intended to do?

---

That's right, and thanks for clarifying, I'll refine it.